projects/components/src/header/page/page-header.component.ts

```diff
@@ -66,8 +72,11 @@ export class PageHeaderComponent implements OnInit {
   @Input()
   public isBeta: boolean = false;
 
-  @Input()
-  public contentAlignment: PageHeaderContentAlignment = PageHeaderContentAlignment.Column;
+  @ContentChild(HeaderContentPrimaryDirective, { static: true })
+  public contentPrimary!: HeaderContentPrimaryDirective;
```

Comment:
these two should be optional, private and readonly

Reply:
kk, why private ?